src/cli/commands/object/get.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const Command = require('ronin').Command
+const utils = require('../../utils')
+const bs58 = require('bs58')
+const debug = require('debug')
+const log = debug('cli:object')
+log.error = debug('cli:object:error')
+
+module.exports = Command.extend({
+  desc: 'Get and serialize the DAG node named by <key>',
+
+  options: {},
+
+  run: (key) => {
+    var ipfs = utils.getIPFS()
+
+    if (!key) {
+      throw new Error("Argument 'key' is required")
+    }
+
+    if (utils.isDaemonOn()) {
+      return ipfs.object.get(key, (err, obj) => {
+        if (err) {
+          log.error(err)
+          throw err
+        }
+
+        console.log(JSON.stringify(obj))
```

Comment:
does this print in the same way that go-ipfs does? I believe go-ipfs doesn't some formatting rather then printing the obj directly